modules/serverless-workflow-traffic-splitting.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="traffic-splitting_{context}"]
+= Traffic splitting using Knative Client
+
+`kn` helps you control which revisions get routed traffic on your Knative service.
+
+Knative service allows for traffic mapping, which is the mapping of revisions of the service to a allocated portion of traffic. It offers the option to create unique URLs for particular revisions and has the ability to assign traffic to the latest revision.
+
+With every update to the configuration of the service, a new revision is created with the service route pointing all the traffic to the latest ready revision by default.
+
+You can change this behavior by defining which revision gets a portion of the traffic.
+
+For example, to route 10% of traffic to your new revision before putting all traffic on, use the command: 
+
+----
```

Comment:
And again for this portion, I'd try to make this a traditional procedure module style.